src/main/java/HelloWorld.java

```diff
@@ -3,6 +3,7 @@
     public static void main(String[] args) {
 	String myTimeString = "test error";
 	String myTimeString1 = "test error";
+	String myTimeString2 = "test error";
 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://77.121.149.190:9000/coding_rules#rule_key=squid%3AS106)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "hello world" 13 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://77.121.149.190:9000/coding_rules#rule_key=squid%3AS1192)